robot/install.bash

```diff
@@ -1,4 +1,23 @@
-#! /usr/bin/env bash
+#!/bin/bash
+
+if [[ "$USER" != "amigo" ]]
+  then
+    read -r -p "This package ($target) should only be installed on the robot itself, not on your own PC. Are you sure to continue? [y/N] " response
+    response=${response,,}    # tolower
+    if [[ "$response" =~ ^(yes|y)$ ]]
+    then
+      :
+    else
+      return 1
+    fi
+fi
+
+#Do not require sudo
+if [ ! -f /etc/sudoers.d/sudo-no-password ]
+then
+    echo '%sudo ALL=(ALL) NOPASSWD:ALL' | sudo tee /etc/sudoers.d/sudo-no-password && \
+    sudo chmod 440 /etc/sudoers.d/sudo-no-password
+fi
```

Comment:
I don't like this. This reduces protection against idiotes.

Reply:
Sure, can be removed.